e2e-tests/cypress/e2e/journeys/insurance/check-your-answers/need-to-start-new-application/need-to-start-new-application-page.spec.js

```diff
@@ -84,7 +85,7 @@ context('Insurance - Check your answers - Need to start new application page', (
     });
 
     it('renders a `start new application` link button', () => {
-      cy.checkLink(linkButtons.startNewApplication(), START, BUTTONS.START_A_NEW_APPLICATION);
+      cy.checkLink(linkButtons.startNewApplication(), BUYER_COUNTRY, BUTTONS.START_A_NEW_APPLICATION);
```

Comment:
Do we have an E2E test that covers starting a new application after submitting one?

Reply:
yep. `e2e-tests\cypress\e2e\journeys\insurance\check-your-answers\need-to-start-new-application\need-to-start-new-application-start-new-application.spec.js`

Reply:
Fantastic